web/app/themes/gds/app/Providers/WooCommerceServiceProvider.php

```diff
@@ -147,4 +148,19 @@ public function filterFeatures(array $features): array
 
         return $features;
     }
+
+    /* Change price display for Variable Products */
+    public function changeVariablePriceDisplay($price, $productObj)
+    {
+        if ($productObj->get_type() !== 'variable') {
+            return $price;
+        }
+
+        $minPrice = $productObj->get_variation_price('min', true);
+
+        // Translators: %s is the lowest variation price.
+        $price = sprintf(__('Starting from: %s', 'your-text-domain'), wc_price($minPrice));
```

Comment:
```suggestion
        $price = sprintf(__('Starting from: %s', 'gds'), wc_price($minPrice));
```

and you can remove the translation comment above